hyp3lib/rtc2color.py

```diff
@@ -107,8 +136,12 @@ def rtc2color(fullpolFile, crosspolFile, threshold, geotiff, cleanup=False,
   if real == True:
     red = (2.0*rp*(1 - blue_mask) + zp*blue_mask)
   else:
+    # FIXME: Should we *scale* this? There are values over 1
     red = (2.0*rp*(1 - blue_mask) + zp*blue_mask)*255
+  # FIXME: We should be *adding* 1 I think... values less than 1 will be between
+  #        the no data value and our minimum valid value...
   red[red==0] = 1
+  # FIXME: Is this the right mask (xp > 0)? Red should be mostly copol
   red = red * mask
```

Comment:
>     red: simple bounce (polarized) with some volume scattering

This is attempting to mask our all no data and cleaned values, but only uses crosspol to define that. Red should be mostly copol, with a crosspol component. Does it make sense to mask based on crosspol only?

Reply:
This is the right mask as long as the cleanup threshold is << than the user selected power threshold.